.mergify.yml

```diff
@@ -110,6 +114,36 @@ pull_request_rules:
         strict: false
       dismiss_reviews: {}
       delete_head_branch: {}
+  # release-1.6 branch
+  - name: automerge backport release-1.6
+    conditions:
+      - author=mergify[bot]
+      - base=release-1.6
+      - label!=do-not-merge
+      - 'status-success=DCO'
+      - 'check-success=canary'
+      - 'check-success=unittests'
+      - 'check-success=golangci-lint'
+      - 'check-success=codegen'
+      - 'check-success=lint'
+      - 'check-success=modcheck'
+      - 'check-success=pvc'
+      - 'check-success=pvc-db'
+      - 'check-success=pvc-db-wal'
+      - 'check-success=encryption-pvc'
+      - 'check-success=encryption-pvc-db'
+      - 'check-success=encryption-pvc-db-wal'
+      - 'check-success=encryption-pvc-kms-vault-token-auth'
+      - 'check-success=TestCephSmokeSuite'
+      - 'check-success=TestCephHelmSuite'
+      - 'check-success=TestCephMultiClusterDeploySuite'
+      - 'check-success=TestCephUpgradeSuite'
```

Comment:
I'm not sure how to validate that these are correct. I think they should be, but I don't know for sure. Is there a mergify debug tool I'm not finding?

Reply:
You use the job's name which seems correct to me.